xilem_web/src/concurrent/memoized_await.rs

```diff
@@ -148,7 +291,7 @@ impl MemoizedAwaitState {
 }
 
 #[derive(Debug)]
-enum MemoizedAwaitMessage<Output: std::fmt::Debug> {
+enum MemoizedAwaitMessage<Output: fmt::Debug> {
```

Comment:
Maybe a different name here as well, as it's not just for the `MemoizedAwait` anymore?